packages/ibm-products/src/global/js/hooks/useResizeObserver.ts

```diff
@@ -95,8 +95,7 @@ export const useResizeObserver = (
     return () => {
       observer.disconnect();
     };
-    // Ignoring exhaustive-deps as we do NOT want to include the ref in dep array
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [ref.current]);
```

Comment:
I noticed that removing this from the dependency array causes the hook to return `0` for width upon the initial resize observer update, thanks to Percy flagging a visual change in the stacked tearsheets 😄